redisgears_core/src/lib.rs

```diff
@@ -1005,6 +1002,9 @@ macro_rules! get_allocator {
 #[allow(missing_docs)]
 mod gears_module {
     use super::*;
+    use config::{GEARS_BOX_ADDRESS, LIBRARY_MAX_MEMORY, REMOTE_TASK_DEFAULT_TIMEOUT};
+    use rdb::REDIS_GEARS_TYPE;
+    use redis_module::{configuration::ConfigurationFlags, redis_command};
```

Comment:
I do not think `redis_command` is needed, is it?

Reply:
I think I fixed it no?
https://github.com/RedisLabsModules/redismodule-rs/blob/308a3ef9f1ad07c294dea27e30d7170db4f9f593/src/macros.rs#L224